tests/bitset.test.cpp

```diff
@@ -0,0 +1,202 @@
+#include <boost/ut.hpp>
+#include <cstdint>
+#include <libxbitset/bitset.hpp>
+
+namespace embed {
+boost::ut::suite bitset_test = [](){
+    using namespace boost::ut;
+
+    //TEST SET() FOR INITIAL VALUE OF 1
+    "bitset::set()_set_no_change_1"_test = [](){
+        xstd::bitset<int> test_set(1); 
```

Comment:
```suggestion
        xstd::bitset test_set(1); 
```

You shouldn't need the template here